vendor/ocaml-4.12/clflags.mli

```diff
@@ -0,0 +1,272 @@
+(**************************************************************************)
```

Comment:
We don't need the whole `Clflags`, we just need some of the variables and I think we want to decide their value instead of using the defaults.

Reply:
done

Reply:
Perhaps this module should be moved out of this library to make it clear that it's a stubs and not the origin source. For example in the `parser-shims` new library.

Reply:
done